Sources/MaliciousSiteProtection/Services/EmbeddedDataProvider.swift

```diff
@@ -58,8 +58,7 @@ public class PhishingDetectionDataProvider: PhishingDetectionDataProviding {
              let filterSetData = try loadData(from: embeddedFilterSetURL, expectedSHA: embeddedFilterSetDataSHA)
              return try JSONDecoder().decode(Set<Filter>.self, from: filterSetData)
          } catch {
-             Logger.phishingDetectionDataProvider.error("🔴 Error: SHA mismatch for filterSet JSON file. Expected \(self.embeddedFilterSetDataSHA)")
-             return []
+             fatalError("🔴 Error: SHA mismatch for filterSet JSON file. Expected \(self.embeddedFilterSetDataSHA)")
```

Comment:
I know we do this for privacy configs, but I'm not a fan of crashing the app in this instance. However, ideally this would never make it past internal testing so I'm okay with it.

Reply:
Reaching the `catch` block means the app bundle corruption (same is valid for a missing Storyboard), and considering the [Let it crash](https://app.asana.com/0/1199230911884351/1199550018748895/f) discussion this makes impossible to run the app further and we must crash here.
Plus, it happens early in the app lifetime during smoke testing that makes it easily noticeable and not affecting the production built.